testcases/PowerNVDump.py

```diff
@@ -370,6 +370,8 @@ def kernel_crash(self, crash_type="echo_c"):
         elif crash_type == "hmc":
             self.cv_HMC.run_command("chsysstate -r lpar -m %s -n %s -o dumprestart" %
                                    (self.system_name, self.lpar_name), timeout=300)
+        elif crash_type == "lock":
+            self.c.pty.sendline("insmod /tmp/kernel-module-bug-196448/softlockup.ko")
```

Comment:
module should be in generic folder, exlpre op-test/test_binaries  to keep all your c files

Reply:
Done